src/test/server/apis/outgoing-prs-test.ts

```diff
@@ -309,14 +316,15 @@ test('[outgoing-prs-test]: with success status', (t) => {
       allow_squash_merge: true,
       allow_merge_commit: true,
     },
-    mergeable: MergeableState.UNKNOWN,
+    mergeable: MergeableState.MERGEABLE,
```

Comment:
Why did this change?

Reply:
I *believe* it's because we are using an actual github token for project-health1 where as before it was just public information.